src/renderer/components/siteinfotools/ProviderDropdown.tsx

```diff
@@ -111,8 +117,9 @@ export const ProviderDropdown = (props: Props) => {
 				{multiMachineSelect ? 'Create new site from' : 'Back up to'}
 			</span>
 			<FlyDropdown
-				className={styles.ProviderDropdown}
+				className={`${styles.ProviderDropdown} ${offline ? styles.ProviderDropdownOffline : ''}`}
 				classNameList={styles.ProviderDropdown_List}
+				classNameListItem={offline ? styles.DropdownItemOffline : ''}
```

Comment:
Can you use `classnames` here too? 
```suggestion
				classNameListItem={classnames([styles.DropdownItemOffline]: offline )}
```